docs/content/latest/quick-start/build-apps/go/ysql-pg.md

```diff
@@ -0,0 +1,351 @@
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../ysql-pgx/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PGX
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pq/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PQ
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-pg/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - PG
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-gorm/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - GORM
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+</ul>
+
+The following tutorial creates a simple Go application that connects to a YugabyteDB cluster using the [Go-pg client and ORM](https://pkg.go.dev/github.com/go-pg/pg), performs a few basic database operations — creating a table, inserting data, and running a SQL query — and then prints the results to the screen.
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you are new to YugabyteDB, you can download, install, and have YugabyteDB up and running within minutes by following the steps in [Quick start](../../../../quick-start/). Alternatively, you can use [Yugabyte Cloud](http://cloud.yugabyte.com/register), to get a fully managed database-as-a-service (DBaaS) for YugabyteDB.
+
+- [Go version 1.8](https://golang.org/dl/), or later, is installed.
+
+### SSL/TLS configuration (Optional)
+
+You can choose to enable or disable SSL for your local YugabyteDB cluster. Refer [here](../../../../secure/tls-encryption/client-to-server/) to learn about the SSL/TSL configuration needed for setting up your YugabyteDB cluster. In case of Yugabyte Cloud, you will have SSL enabled when you create a cluster since it's pre-configured with SSL/TLS enabling client-side authentication.
+
+#### CA certificate
+
+Create a [CA certficate](../../../../secure/tls-encryption/server-certificates/#generate-the-root-certificate-file) to launch your YugabyteDB cluster with SSL/TLS enabled.
+
+In case of a Yugabyte Cloud cluster, to download the CA certificate for your cluster in Yugabyte Cloud, do the following:
+
+1. On the **Clusters** tab, select a cluster.
+
+1. Click **Connect**.
+
+1. Click **Connect to your application** and download the CA cert.
+
+#### OpenSSL
+
+Install [OpenSSL](https://www.openssl.org/) 1.1.1 or later only if you have a YugabyteDB setup with SSL/TLS enabled. Yugabyte Cloud clusters are always SSL/TLS enabled.
+
+Here is the table summarizing the SSL modes and its support in the driver:
+
+| SSL Mode | Client driver behavior |
+| :--------- | :---------------- |
+| disable | Supported |
+| allow | Supported |
+| prefer | Supported |
+| require | Supported |
+| verify-ca | Supported |
+| verify-full | Supported |
```

Comment:
Yugabyte Cloud requires SSL/TLS, and connections using SSL mode disable will fail.